@navikt/aksel-stylelint/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "@navikt/aksel-stylelint",
+  "version": "3.0.0-beta.3",
```

Må synkes med resten av pakkene (tror det er 3.3.1 nå)
```suggestion
  "version": "3.3.1",
```